search/includes/classes/queue/class-cron.php

```diff
@@ -62,6 +62,8 @@ public function init() {
 		add_action( self::SWEEPER_CRON_EVENT_NAME, [ $this, 'sweep_jobs' ] );
 		add_action( self::TERM_UPDATE_CRON_EVENT_NAME, [ $this, 'queue_posts_for_term_taxonomy_id' ] );
 
+		add_filter( 'a8c_cron_control_concurrent_event_whitelist', [ $this, 'configure_concurency' ] );
```

Comment:
The filter has to be registered before  / up-to the `after_setup_theme` hook: https://github.com/Automattic/Cron-Control/blob/cde4f55e1b129fca22e27ad1310f9a2f7021e389/includes/class-events.php#L51

I'm _pretty_ sure we're good here, just noting.